### PR TITLE
Trish txt july6

### PR DESCRIPTION
**Pull request by trishreimer**
minor text updates to the help pg.

### CODE REVIEW

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://business-ar-dev--pr-265-vs3ccret.web.app

File: web/site/content/en-CA/help.md [hunk 1]
```diff
@@ -1,33 +1,43 @@
...
+- Setup your <a href="https://id.gov.bc.ca/account/" target="_blank">BC Services Card (BCSC) Account</a>. This digital ID is a secure and easy way to login to access government services. 
+- Review and confirm the registered office addresses and directors in Corporate Online.
+- If updates are needed, go to <a href="https://www.corporateonline.gov.bc.ca" target="_blank">Corporate Online</a> to update, then return to file your annual report.
+- When you file your annual report, a Basic account will be created. <a href ="https://www2.gov.bc.ca/gov/content/employment-business/business/managing-a-business/permits-licences/news-updates/modernization-updates/using-the-business-registry#account"> Learn more about a Basic account </a> to ensure it is right for you. 
+- If you need a <a href ="https://www2.gov.bc.ca/gov/content/employment-business/business/managing-a-business/permits-licences/news-updates/modernization-updates/using-the-business-registry#account">Premium account</a>, set it up <a href = "https://www.account.bcregistry.gov.bc.ca/choose-authentication-method">here </a> then use the link in your Annual report reminder to complete your annual report filing.
```
Comment (thorwolpert) [inline web/site/content/en-CA/help.md:8]: There's a mixture of opening a new tab, or going to the linked page.
It looks deliberate, as all the account setup pages are direct links

**Review by thorwolpert — COMMENTED** [review]
Just a question on link interaction style.

**Comment by thorwolpert** [comment]
https://business-ar-dev--pr-265-vs3ccret.web.app/en-CA

**Pull request closed without merging**